docs/software/cloud.md

```diff
@@ -192,9 +193,10 @@ Nextcloud gives you access to all your files wherever you are. Store your docume
         bantime = 600
         ```
 
-        Assure that the `logpath` matches the `'datadirectory'` value in the Nextcloud configuration file (`config.php` see above), or the `'logfile'` value if defined.
+        Assure that the `logpath` matches the `'datadirectory'` value in the Nextcloud configuration file (`config.php`, see above), or the `'logfile'` value, if defined.
 
-        Properties not defined here are taken from the `[DEFAULT]` block in `/etc/fail2ban/jail.conf` or `/etc/fail2ban/jail.local`, if present. Note the setting `backend = auto`: By default, `backend` is set to `systemd` in `/etc/fail2ban/jail.conf`. As a result, Fail2Ban would ignore the `logpath` entry here in the jail `nextcloud.conf`, with the consequence, that Fail2Ban does not recognize an attack on Nextcloud (port 80, 443) even though attacks are logged in `/mnt/dietpi_userdata/nextcloud_data/nextcloud.log`.
+        Properties not defined here are taken from the `[DEFAULT]` block in `/etc/fail2ban/jail.conf` or `/etc/fail2ban/jail.local`, if present. Note the setting `backend = auto`. By default, `backend` is set to `systemd` in `/etc/fail2ban/jail.conf`.  
+        As a result, Fail2Ban would ignore the `logpath` entry here in the jail `nextcloud.conf`, with the consequence, that Fail2Ban does not recognize an attack on Nextcloud (port 80, 443), even though attacks are logged in `/mnt/dietpi_userdata/nextcloud_data/nextcloud.log`.
```

Comment:
If using a line break at all, doesn't it make more sense before "Note the setting" to have this block with belongs together also in one paragraph?